src/test/java/com/serenitydojo/WhenWorkingWithFloatingPointNumbers.java

```diff
@@ -0,0 +1,31 @@
+package com.serenitydojo;
+
+import org.junit.Test;
+
+import java.io.FilterOutputStream;
+
+public class WhenWorkingWithFloatingPointNumbers {
+    @Test
+// test to convert a number from celsius to fahrenheit
+    public void convertCelsiusToFahrenheit(){
+
+        double celsius = 50.0;
+        double fahrenheit = 0.0;
```

Comment:
initializing with 0 doesn't really do anything here. cleaner code would be to directly assign the value we need like  
```suggestion
        double fahrenheit = (celsius * 1.8) + 32;
```

Reply:
Thanks you for the feedback